specification/wse/src/main/scripts/org/kaazing/specification/wse/opening/request.header.x.websocket.protocol/handshake.request.rpt

```diff
@@ -33,6 +33,6 @@ read version "HTTP/1.1"
 read header "X-WebSocket-Protocol" /primary|secondary/
 read header "Content-Type" /text\/plain;charset=(?i)utf-8/
 
-read "http://localhost:8080/path/uofdbnreiodfkbqi\n"
-read "https://localhost:8443/path/kwebfbkjwehkdsfa\n"
+read /http:\/\/localhost:8080\/path\/.+?\n/
+read /http:\/\/localhost:8080\/path\/.+?\n/
```

Comment:
.+?\n would be two or more characters which is not what is intended (+ is one or more, * is zero or more). I settled for 
read "http://localhost:8080/path/" /.+\n/